gossip/hub/instance.go

```diff
@@ -171,6 +171,9 @@ func SetUpInstance(ctx context.Context, client trillian.TrillianLogClient, cfg *
 		switch src.HashAlgorithm {
 		case sigpb.DigitallySigned_SHA256:
 			hasher = crypto.SHA256
+		case sigpb.DigitallySigned_NONE:
```

Comment:
Might be better if we pair up the valid combinations of signature and hash but not in scope for this really.

Reply:
That might be nice but I don't think it would belong here &ndash; knowledge of crypto algorithm compatibility should be buried in some `trillian/mumble/crypto` package somewhere.

Reply:
I think we've done it elsewhere e.g. the map code.